### PR TITLE
Added Docker Extension to feature carousel

### LINKED ISSUE #2706: [Meshery] Add Docker Extension to feature carousel
**Current Behavior**
A new Docker Extension for Meshery is being released. See details: https://layer5.io/docker-extension-meshery.

The feature carousel on https://layer5.io/service-mesh-management/meshery does not include a section for this extension, but should.

**Implementation**
Add screenshot, description, and link to extension page in the feature carousel.

---
**Screenshots / Mockups**

Screen Shot 2022-04-10 at 8 41 36 AM

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Abhijay007**
Signed-off-by: Abhijay Jain <Abhijay007j@gmail.com>

**Description**

Added Docker Extension to feature carousel

This PR fixes #2706 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 149284486d12792b55af827df5833416c4dfcd23 at: https://625faa1f3da68c478c21b8dc--layer5ng.netlify.app

File: src/sections/Meshery/Meshery-features/index.js [hunk 1]
```diff
@@ -238,15 +261,44 @@ const MesheryFeatures = () => {
...
+                  </a>
+                  <p>
+                    The Docker Extension for Meshery extends Docker Desktop’s
+                    position as the cloud native developer’s go-to Kubernetes
+                    environment with easy access to the next layer of cloud
+                    native infrastructure: service meshes.
+                  </p>
+                </div>
+              ),
+              learnMoreLink: "https://layer5.io/docker-extension-meshery",
```
Comment (debo19) [inline src/sections/Meshery/Meshery-features/index.js:300]:
```suggestion
              learnMoreLink: "/docker-extension-meshery",
```

File: src/sections/Meshery/Meshery-features/index.js [hunk 2]
```diff
@@ -238,15 +261,44 @@ const MesheryFeatures = () => {
...
               ),
               learnMoreLink: "https://docs.meshery.io/extensibility",
             },
+            {
+              title: "Docker Extension for Meshery",
+              description:
+                "Develop service mesh-ready apps using Docker Desktop and Meshery",
+              content: (
+                <div className="feature-expose-full">
+                  <a href={img7}>
```
Comment (debo19) [inline src/sections/Meshery/Meshery-features/index.js:286]:
```suggestion
                 
```
This `<a>` tag can be removed.

Comment (debo19) [reply src/sections/Meshery/Meshery-features/index.js:286]: You can remove it from the previous feature as well.

**Review by debo19 — CHANGES_REQUESTED** [review]

**Comment by Abhijay007** [comment]
Thanks @debo19, for suggestions, will do that :)

**Comment by l5io** [comment]
🚀 Preview for commit d0f4d0c47b1b4185ad1c8425b1575c5b1977b892 at: https://62600d0eae9611200f943b37--layer5ng.netlify.app

**Review by debo19 — APPROVED** [review]
LGTM 🎉  Thanks @Abhijay007

**Pull request merged**